data/courses.yaml

```
@@ -18,6 +18,21 @@
 # 
 # This list is not ordered. Add new entries wherever you want.
 
+- name: Introduction to formal verification of mathematics
+  instructor: Philip Matchett Wood
+  institution: Harvard University
+  lean_version: 3
+  tags: ['mathematics']
```

```suggestion
  tags: ['mathematics']
  material: https://leanprover-community.github.io/mathematics_in_lean/mathematics_in_lean.pdf
```
It's not necessary, but do you have a course website or repository you want to include?